metals/src/main/scala/scala/meta/internal/metals/MetalsLogger.scala

```diff
@@ -88,21 +89,8 @@ object MetalsLogger {
     }
   }
 
-  def newBspLogger(workspace: AbsolutePath): Logger = {
-    val logfile = workspace.resolve(workspaceLogPath)
-    Logger.root
-      .orphan()
-      .clearModifiers()
-      .clearHandlers()
-      .withHandler(
-        writer = newFileWriter(logfile),
-        formatter = defaultFormat,
-        minimumLevel = Some(Level.Info)
-      )
-  }
-
   def newFileWriter(logfile: AbsolutePath): FileWriter =
-    FileWriter().path(_ => logfile.toNIO).autoFlush
+    FileWriter(pathBuilder = PathBuilder.static(logfile.toNIO)).flushAlways
```

Comment:
`autoFlush` here was `flushAlways`. There is a new asynchronous flush that wasn't available before, but I just stuck with what we had before and se flush to always.